CMakeLists.txt

```diff
@@ -543,6 +544,7 @@ if (LWS_WITH_ZLIB)
 			foreach (lib ${LWS_LIBRARIES})
 				add_dependencies(${lib} zlib_internal)
 			endforeach()
+			list(APPEND LWS_LIBRARIES zlib_internal)
```

Comment:
This should not be needed. We're linking to zlib statically. `LWS_LIBRARIES` is only used for exporting the libwebsocket libraries to other cmake projects.

So this line can be removed